docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -95,12 +95,13 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
      (product, warehouse, location), (warehouse), (product), (warehouse, product), ())`.
     The N elements of a `CUBE` specification results in 2^N `GROUPING SETS`.
 
-* **Mixed Grouping Analytics**
+* **Mixed/Nested Grouping Analytics**
```

Comment:
I feel `Nested Grouping Analytics` looks confusing because `GROUPING SETS` only accepts the nested cases.

Reply:
> I feel `Nested Grouping Analytics` looks confusing because `GROUPING SETS` only accepts the nested cases.

So let's make these two case separated?

Reply:
IMO the nested case is a kind of the mixed cases, so we don't need to explicitly say it here.

Reply:
Ah, okay. on second thought, `**Mixed/Nested Grouping Analytics**` looks okay (I couldn't come up with a better title...)